core/iwasm/include/wasm_export.h

```diff
@@ -289,6 +295,12 @@ typedef struct wasm_global_inst_t {
     void *global_data;
 } wasm_global_inst_t;
 
+/* Memory instance*/
+typedef struct wasm_memory_inst_t {
+    void *address;
+    uint32_t size;
+} wasm_memory_inst_t;
```

Comment:
Had better remove this struct since it is unused.

Reply:
Thanks, good catch.